vello/src/scene.rs

```diff
@@ -106,6 +106,37 @@ impl Scene {
         self.encoding.encode_end_clip();
     }
 
+    /// Draw a rounded rectangle blurred with a gaussian filter.
+    pub fn draw_blurred_rounded_rect(
+        &mut self,
+        transform: Affine,
+        brush: Color,
+        size: impl Into<Size>,
```

Comment:
I'd *probably* expect to see this be a `Rect`, to create flexibility in where the origin is (even if internally, we just make it use a transform to get to the current format).

At the moment, it's quite unclear to me where the origin actually is, except for by looking at the example